src/rnr_score_hv/innov_netcdf.py

```diff
@@ -322,8 +349,31 @@ def get_data(self):
                 raise ValueError(msg) from err
 
             ncfile.close()
-        
-        # could convert the list of named tuples to a pandas dataframe
-        # df = pd.DataFrame(harvested_data), columns=harvested_data.fields)
 
-        return harvested_data
+        if self.config.get_output_format() == hvs.PANDAS_DATAFRAME:
+            harvested_data_pd = self.get_data_pandas_df(harvested_data)
+            return harvested_data_pd
+
+        return harvested_data 
+
+
+    def get_data_pandas_df(self, data):
```

Comment:
should this belong in a base class?

Reply:
maybe at some point, yes.  However for this prototype I just put it here for starters.